### PR TITLE
refactor: cleanup UI components

### PR DESCRIPTION
**Pull request by resir014**
Use a better foldering structure for reusable components.

- `helpers`: one-time use components (e.g. provider)
- `icons`: any SVG icons exported as React components
- `layouts`: layout-specific components (e.g. `Page`, `Content`)

### CODE REVIEW

File: components/icons/Logo.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
```
Comment (adibfirman) [inline components/icons/Logo.tsx:1]: karena ini file SVG ya, i think gak di taro di confignya chakra ui aja kah?

**Review by adibfirman — COMMENTED** [review]
Cuman nambahin sedikit aja sih

File: components/icons/Logo.tsx (hunk 1 above)
Comment (resir014) [reply components/icons/Logo.tsx:1]: Boleh2. I'll work on that shortly.

Comment (resir014) [reply components/icons/Logo.tsx:1]: @adibfirman Kayaknya agak susah deh. Soalnya logo React ID ini gak square, dan karena komponen `Icon` nge-force svg-nya jadi square, jadi mengecil di nav-nya

Comment (adibfirman) [reply components/icons/Logo.tsx]: i see, good to know mas, oke dari ku sisanya LGTM 💯

**Review by adibfirman — APPROVED** [review]
LGTM 💯, tinggal conflictnya aja 👌

**Pull request merged**